patches/tModLoader/Terraria/Recipe.cs.patch

```diff
@@ -473,8 +488,29 @@
 +			if (Main.recipe[i].Disabled)
 +				continue;
 +
- 			if (!Main.recipe[i].notDecraftable)
- 				ItemID.Sets.IsCrafted[Main.recipe[i].createItem.type] = i;
++			/*Vanilla:
```

Comment:
No need for `Vanilla:`. Block comments are assumed to be vanilla code, particularly if the corresponding tML replacement code comes directly after.

Please note, when commenting code. We only put the `/*` and `*/` on lines surrounding the code. We don't add `* ` to every line in the body, to reduce patches.

Reply:
👍